ucsb-cs156-f24/proj-dining-f24-13
# sj-Create placeholder page for individual dining commons

Closes #30 

This PR Creates a placeholder page for individual dining commons at the address /diningcommons/:diningcommonscode

How to test:
- Run on localhost and log in with any email
- go to http://localhost:8080/diningcommons/carillo and make sure it says "Placeholder for Dining Commons Page for carillo"
- go to http://localhost:8080/diningcommons/dlg and make sure it says "Placeholder for Dining Commons Page for dlg"

<img width="1425" alt="Screenshot 2024-11-19 at 10 18 04 PM" src="https://github.com/user-attachments/assets/db7a1f66-cb54-4041-99af-b5ba7a0bd357">
